data/templates/esx-boot-cfg

```diff
@@ -0,0 +1,7 @@
+bootstate=0
+title=Loading ESXi installer
+kernel=<%=tbootFile%>
+kernelopt=runweasel formatwithmbr com1_baud=115200 com1_Port=<%=comportaddress%> tty2Port=<%=comport%> debugLogToSerial=1 logPort=<%=comport%> ks=http://<%=server%>:<%=port%>/api/common/templates/esx-ks
+modules=<%=moduleFiles%>
+build=
```

Comment:
Thanks @zyoung51 , your suggestion simplifies our design. I want to make another task to implement your idea but not in this PR, since it involves on-tasks change and need some testing.